src/Helpers.php

```diff
@@ -156,4 +162,118 @@ public static function cleanSchemaName(string $name): string {
 		global $readableAppID;
 		return substr($name, strlen($readableAppID));
 	}
+
+	protected static function getScopeNameFromAttributeArgument(Arg $arg, string $routeName): ?string {
+		if ($arg->name->name === 'scope') {
+			if ($arg->value instanceof ClassConstFetch) {
+				if ($arg->value->class->getLast() === 'OpenAPI') {
+					return self::getScopeNameFromConst($arg->value);
+				}
+			} elseif ($arg->value instanceof String_) {
+				return $arg->value->value;
+			} else {
+				Logger::panic($routeName, 'Can not interpret value of scope provided in OpenAPI(scope: …) attribute. Please use string or OpenAPI::SCOPE_* constants');
+			}
+		}
+
+		return null;
+	}
+
+	protected static function getScopeNameFromConst(ClassConstFetch $scope): string {
+		return match ($scope->name->name) {
+			'SCOPE_DEFAULT' => 'default',
+			'SCOPE_ADMINISTRATION' => 'administration',
+			'SCOPE_FEDERATION' => 'federation',
+			'SCOPE_IGNORE' => 'ignore',
+			// Fall back for future scopes assuming we follow the pattern (cut of 'SCOPE_' and lower case)
+			default => strtolower(substr($scope->name->name, 6)),
+		};
+	}
+
+	public static function getAttributeScopes(ClassMethod|Class_|Node $node, string $annotation, string $routeName): array {
```

Comment:
`$annotation` is always `OpenAPI`, it could just be skipped from the parameters (and the method name should reflect that).

Reply:
Yeah, just followed the pattern of existing methods, but yeah they are used for different attributes, so we can simplify this